src/corelibs/U2Core/src/util/U2SequenceUtils.h

```diff
@@ -168,15 +168,28 @@ class U2CORE_EXPORT U2SequenceDbiHints {
     static const QString EMPTY_SEQUENCE;
 };
 
-// untwist/expand circular sequence
-class U2CORE_EXPORT U2PseudoCircularization : public QObject {
+/**
+ * Utility class to deal with circular sequences.
+ * Supports mapping of the original sequence coordinates to a circular sequence coordinates with a specified 'overlap' range.
+ */
+class U2CORE_EXPORT U2PseudoCircularization {
+    U2PseudoCircularization() = default;
```

Comment:
If it's a utility class, the constructor should be `delete`'d

Reply:
I do the same here: make it a private.